### ISSUE TITLE
Ability to control envoy log level for a workload, at runtime

### ISSUE THREAD

**Issue by Stono** [istio/istio#51703]
**Describe the feature request**
It would like to be able to set the sidecar log level for a workload at runtime to debug issues.  Currently i'd have to port-forward 15000 to each pod and use the ui or api there to set the log levels.  This is extremely tedious when a workload has many replicas.  It'd be much more preferable to be able to set that via a first class api, for example perhaps the sidecar, as that is ultimately what i'm configuring?  https://istio.io/latest/docs/reference/config/networking/sidecar/.

It'd also then be nice to have a cluster level setting too?

The reason I want this as a runtime thing is because we wouldn't want certain log levels enabled all the time, as they're very noisy, but if we're investigating an acute issue, being able to toggle them on is important.   

**Describe alternatives you've considered**
I know I can use `sidecar.istio.io/componentLogLevel` but that would restart the workload, which can in itself "fix" issues or get rid of the ability to debug a pod _in a broken state_, eg https://github.com/istio/istio/issues/51612.

**Affected product area (please put an X in all that apply)**

[ ] Ambient
[ ] Docs
[ ] Dual Stack
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Extensions and Telemetry
[ ] Security
[ ] Test and Release
[x] User Experience
[ ] Developer Infrastructure

**Affected features (please put an X in all that apply)**

[ ] Multi Cluster
[ ] Virtual Machine
[ ] Multi Control Plane

**Additional context**

**Comment by zirain** [comment]
`istioctl pc log`?

**Comment by Stono** [comment]
Still requires me to do a command for each pod?
And doesn't allow me to do it a the cluster scope (second requirement)

**Comment by zirain** [comment]
.Values.global.proxy.componentLogLevel

**Comment by hzxuzhonghu** [comment]
Write a sxcript and apply to each pod, hha

**Comment by Stono** [comment]
Yes, obviously - i actually do that now.  But i'm making suggestions to improve the user experience of operators working with Istio.
